src/Schema/PlasterManifest-v1.xsd

```diff
@@ -448,6 +448,31 @@
                       <xs:documentation>Specifies the value of the RootModule property.</xs:documentation>
                     </xs:annotation>
                   </xs:attribute>
+                  <xs:attribute name="copyright" type="xs:string">
```

Comment:
Ignore this.  As I mentioned, I had already bumped the schema version for other (unreleased) changes I made to the schema.